igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -57,19 +44,21 @@ class TravisService implements BuildService {
     final String baseUrl
     final String groupKey
     final GithubAuth gitHubAuth
+    final int numberOfRepositories
     final TravisClient travisClient
     final TravisCache travisCache
     final private Set<String> artifactRegexes
     protected AccessToken accessToken
     private Accounts accounts
 
-    TravisService(String travisHostId, String baseUrl, String githubToken, TravisClient travisClient, TravisCache travisCache, Iterable<String> artifactRegexes) {
-        this.groupKey     = "${travisHostId}"
-        this.gitHubAuth   = new GithubAuth(githubToken)
-        this.travisClient = travisClient
-        this.baseUrl      = baseUrl
-        this.travisCache  = travisCache
-        this.artifactRegexes = artifactRegexes == null ? Collections.EMPTY_LIST : new HashSet<>(artifactRegexes)
+    TravisService(String travisHostId, String baseUrl, String githubToken, int numberOfRepositories, TravisClient travisClient, TravisCache travisCache, Iterable<String> artifactRegexes) {
+        this.numberOfRepositories = numberOfRepositories
+        this.groupKey             = "${travisHostId}"
+        this.gitHubAuth           = new GithubAuth(githubToken)
+        this.travisClient         = travisClient
+        this.baseUrl              = baseUrl
+        this.travisCache          = travisCache
+        this.artifactRegexes      = artifactRegexes == null ? Collections.EMPTY_LIST : new HashSet<>(artifactRegexes)
```

Comment:
Do you want `artifactRegexes` to be a List or a Set? Here ut can be both.

Reply:
thanks